mne_bids/dataframe.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+from collections import OrderedDict as odict
+
+
+class DataFrame():
```

Comment:
maybe we can call it `MockDataFrame`? It will make it clear that it's not a full fledged data frame like in pandas?

Reply:
ok, changed :)